### PR TITLE
Create organism and review all created organisms

### PR DESCRIPTION
**Pull request by KseniiaNovinskaia**
1. Organism owner clicks on HOST
Screenshot 2021-02-05 at 22 34 20
2. Redirected to the organism form to be filled
Screenshot 2021-02-05 at 22 35 00
3. After organism is created then redirected to the Dashboard page where organism owner can see all created infections (style still to be added)
Screenshot 2021-02-05 at 22 36 42
4. Also user can navigate across infections created and organisms created on top of the Dashboard page
Screenshot 2021-02-05 at 22 37 53

### CODE REVIEW

**Comment by sabinacuccibar** [comment]
Hi Ksenja! the organism.create form should include also an address input (just address)

File: app/views/organisms/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<%= render 'shared/navbar' %>
+
+<div class="container">
+<%= simple_form_for(@organism) do |f| %>
+  <%= f.input :name %>
+  <%= f.input :body_temperature %>
+  <%= f.input :age %>
+  <%= f.input :species, :collection => %w[human pet plant], prompt: "-- SELECT ONE --" %>
```
Comment (sabinacuccibar) [inline app/views/organisms/new.html.erb:9]: here it should be included an address input tab. Then the geocode gem automatically gets latitude and longitude for the map render

File: app/views/pages/dashboard.html.erb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  <div class="mt-2 ml-10" id="organisms">
+  <br><br>
+  <h5 class="mt-2 ml-10">These are your organisms</h5>
+  <br>
+   <% @organisms.each do |organism| %>
+        <div class="container mb-2 mr-4" >
+          <ul style="list-style: none;">
+            <h1><%= organism.name %></h1>
+            <li> <h2>body temperature:<%= organism.body_temperature %> °C</h2> </li>
+            <li> <h2>age: <%= organism.age %></h2></li>
```
Comment (sabinacuccibar) [inline app/views/pages/dashboard.html.erb]: I guess that also here we should add the organism.address

File: app/controllers/organisms_controller.rb [hunk 3]
```diff
@@ -11,8 +11,25 @@ def show
...
+    @organism.user = current_user
+    authorize @organism
+    @organism.save
+    redirect_to dashboard_path
+  end
+
   private
 
+  def organism_params
+    params.require(:organism).permit(:name, :body_temperature, :age, :species)
```
Comment (sabinacuccibar) [inline app/controllers/organisms_controller.rb]: Here too it should be added address among params required

**Review by sabinacuccibar — APPROVED** [review]

**Comment by KseniiaNovinskaia** [comment]
@Kimon-Haars @sabinacuccibar @jasonheeps 
1. have added address into form
Screenshot 2021-02-06 at 14 53 10
2. Added design to my organisms page
Screenshot 2021-02-06 at 14 53 29

**Review by Kimon-Haars — APPROVED** [review]

**Pull request merged**